src/main/java/at/hannibal2/skyhanni/features/inventory/wardrobe/EstimatedWardrobePrice.kt

```diff
@@ -0,0 +1,39 @@
+package at.hannibal2.skyhanni.features.inventory.wardrobe
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.config.ConfigUpdaterMigrator
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class EstimatedWardrobePrice {
+
+    private val config get() = SkyHanniMod.feature.inventory.estimatedItemValues
+
+    @SubscribeEvent
+    fun onTooltip(event: LorenzToolTipEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!config.armor) return
+        if (!WardrobeAPI.inWardrobe()) return
+        if (WardrobeAPI.inCustomWardrobe) return
```

Comment:
"Show Armor Value" is a feature toggle. this check makes it dependent on custom scoreboard. do we this feature want to only work in custom scoreboard, or do we want to keep the original, standalone feature if custom scoreboard is disabled?